### PR TITLE
fix(DB/gameobject): instant spawn on 61-64 quest interactables

### PR DESCRIPTION
**Pull request by BeefSteaker**
## Changes Proposed:
-  fixes a bunch of quest interactables that shouldnt dissapear on looting
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13588
- Closes https://github.com/chromiecraft/chromiecraft/issues/4336

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13672
- Closes https://github.com/chromiecraft/chromiecraft/issues/4387

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13667
- Closes https://github.com/chromiecraft/chromiecraft/issues/4384

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13621
- Closes https://github.com/chromiecraft/chromiecraft/issues/4354
 
## Tests Performed:

- used gm commandts to test on my local
- 

## How to Test the Changes:

1. 
.quest add 9376
.go xyz -870 4397 72
2.
.quest add 10877
.tele terok
.go xyz -3750.310059 4711.770020 -17.320499
3.
.quest add 10881
.tele terok
.go xyz -2843.110107 5350.000000 -30.438601
.go xyz -2860.850098 5482.500000 -30.189301
.go xyz -2679.229980 5403.160156 **-36.469002**
4.
.quest add 9788
.tele funggorcavern
.go xyz -1345 5931 12

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by BeefSteaker** [comment]
Hope its ok to put these all in one PR, its pretty much the same fix for them all

**Comment by Gultask** [comment]
It's ok, don't worry.

**Comment by Gultask** [comment]
Tested. All of them work fine.

**Review by Gultask — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1667666190202265500.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- Quest: A Pilgrim's Plight
+UPDATE `gameobject` SET `spawntimesecs`=0 WHERE `guid`=25340 AND `id`=184478;
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1667666190202265500.sql:2]: Worldserver error: 
Table `gameobject` has gameobject (GUID: 25340 Entry: 184478) with `spawntimesecs` (0) value, but the gameobejct is marked as despawnable at action.

```suggestion
UPDATE `gameobject` SET `spawntimesecs`=1 WHERE `guid`=25340 AND `id`=184478;
```

**Review by Gultask — CHANGES_REQUESTED** [review]

**Review by Gultask — APPROVED** [review]

**Pull request merged**